pages/expandable-section/permutations.page.tsx

```diff
@@ -149,6 +149,20 @@ const permutations = createPermutations<ExpandableSectionProps>([
     headerActions: [<Button variant="inline-link">Some action</Button>],
     children: ['Sample content'],
   },
+  {
+    disablePaddings: [true],
+    defaultExpanded: [false, true],
+    variant: ['default', 'footer', 'navigation', 'stacked', 'container'],
+    headerText: ['No paddings (headerText)'],
+    children: ['Sample content'],
+  },
+  {
+    disablePaddings: [true],
+    defaultExpanded: [false, true],
+    variant: ['default', 'footer', 'navigation', 'stacked', 'container'],
+    header: ['No paddings (header)'],
+    children: ['Sample content'],
+  },
```

Comment:
What's is the visual effect of `disabledPaddings=true` in **No paddings (headerText)**?

<img width="440" alt="image" src="https://github.com/user-attachments/assets/b6b5ebc6-7b05-42d0-ab95-694e737f77d4">